docker/ansible/README.md

```diff
@@ -30,3 +30,6 @@ Once the service is initiated, it will run the `deploy.yml` playbook at the inte
 
 - `deploy_email_address` - The email address that the emails will be sent from
 - `notify_email_address` - The email address that the emails will be sent to
+- `project_directory` - The directory on the local machine that the project is located
+
+**Note**: The `project_directory` variable is used as a convienience for developers, but the `viame-deploy.service` file still contains a hardcoded reference to `home/viame`. If trying to use the service/timer in a local setup under a different directory, you will need to edit the `viame-deploy.service` file by hand.
```

Comment:
could the playbook `systemd.yml`  not also use `.vars.yml` in order to avoid this inconsistency?

Reply:
It could, and I will probably update it to do so later, but it would require me to restructure how the service file works, and I didn't think it'd be immediately worth it to do that, since I'd have to spend more time testing it to make sure it works properly. I'd rather have our build fixed sooner, and that addition is not strictly necessary short term.